### PR TITLE
Format parameter for `CredentialsContainer.create()` & `CredentialsContainer.get()`

### PR DESCRIPTION
**Pull request by skyclouds2001**
### Description

update parameter format for `CredentialsContainer.create()` & `CredentialsContainer.get()` to improve readability

see https://developer.mozilla.org/en-US/docs/Web/API/CredentialsContainer/get and https://developer.mozilla.org/en-US/docs/Web/API/CredentialsContainer/create for current format

### CODE REVIEW

**Comment by skyclouds2001** [comment]
wait for https://github.com/mdn/content/pull/31658 to done first

**Comment by hamishwillee** [comment]
FYI @skyclouds2001 - https://github.com/mdn/content/pull/31658   just got merged.

File: files/en-us/web/api/credentialscontainer/create/index.md [hunk 1]
```diff
@@ -32,20 +32,18 @@ create(options)
...
-        - `password`
-          - : An object containing requirements for creating a password credential. See the [Credential Management API](#credential_management_api) section below for more details.
-        - `publicKey`
-          - : An object containing requirements for creating a public key credential. Causes the `create()` call to request that the user agent creates new credentials via an authenticator — either for registering a new account or for associating a new asymmetric key pair with an existing account. See the [Web Authentication API](#web_authentication_api) section below for more details.
-
     - `signal` {{optional_inline}}
       - : An {{domxref("AbortSignal")}} object instance that allows an ongoing `create()` operation to be aborted. An aborted operation may complete normally (generally if the abort was received after the operation finished) or reject with an "`AbortError`" {{domxref("DOMException")}}.
 
+    Each of the following properties represents a _credential type_ being created, only one of them can be specified:
```
Comment (skyclouds2001) [inline files/en-us/web/api/credentialscontainer/create/index.md:38]:
```suggestion
    - : Each of the following properties represents a _credential type_ being created, only one of them can be specified:
```

File: files/en-us/web/api/credentialscontainer/get/index.md [hunk 2]
```diff
@@ -60,6 +50,19 @@ get(options)
 
       - : An {{domxref("AbortSignal")}} object instance that allows an ongoing `get()` operation to be aborted. An aborted operation may complete normally (generally if the abort was received after the operation finished) or reject with an "`AbortError`" {{domxref("DOMException")}}.
 
+    Each of the following properties represents a _credential type_ being requested:
```
Comment (skyclouds2001) [inline files/en-us/web/api/credentialscontainer/get/index.md:53]:
```suggestion
    - : Each of the following properties represents a _credential type_ being requested:
```

File: files/en-us/web/api/credentialscontainer/get/index.md [hunk 3]
```diff
@@ -60,6 +50,19 @@ get(options)
 
       - : An {{domxref("AbortSignal")}} object instance that allows an ongoing `get()` operation to be aborted. An aborted operation may complete normally (generally if the abort was received after the operation finished) or reject with an "`AbortError`" {{domxref("DOMException")}}.
 
+    - : Each of the following properties represents a _credential type_ being requested:
```
Comment (skyclouds2001) [inline files/en-us/web/api/credentialscontainer/get/index.md:53]:
```suggestion
      Each of the following properties represents a _credential type_ being requested:
```

File: files/en-us/web/api/credentialscontainer/create/index.md [hunk 4]
```diff
@@ -32,20 +32,18 @@ create(options)
...
-        - `password`
-          - : An object containing requirements for creating a password credential. See the [Credential Management API](#credential_management_api) section below for more details.
-        - `publicKey`
-          - : An object containing requirements for creating a public key credential. Causes the `create()` call to request that the user agent creates new credentials via an authenticator — either for registering a new account or for associating a new asymmetric key pair with an existing account. See the [Web Authentication API](#web_authentication_api) section below for more details.
-
     - `signal` {{optional_inline}}
       - : An {{domxref("AbortSignal")}} object instance that allows an ongoing `create()` operation to be aborted. An aborted operation may complete normally (generally if the abort was received after the operation finished) or reject with an "`AbortError`" {{domxref("DOMException")}}.
 
+      Each of the following properties represents a _credential type_ being created, only one of them can be specified:
```
Comment (skyclouds2001) [inline files/en-us/web/api/credentialscontainer/create/index.md:38]:
```suggestion
    Each of the following properties represents a _credential type_ being created, only one of them can be specified:
```

File: files/en-us/web/api/credentialscontainer/get/index.md [hunk 5]
```diff
@@ -60,6 +50,19 @@ get(options)
 
       - : An {{domxref("AbortSignal")}} object instance that allows an ongoing `get()` operation to be aborted. An aborted operation may complete normally (generally if the abort was received after the operation finished) or reject with an "`AbortError`" {{domxref("DOMException")}}.
 
+      Each of the following properties represents a _credential type_ being requested:
```
Comment (skyclouds2001) [inline files/en-us/web/api/credentialscontainer/get/index.md:53]:
```suggestion
    Each of the following properties represents a _credential type_ being requested:
```

File: files/en-us/web/api/credentialscontainer/create/index.md (hunk 1 above)
Comment (wbamberg) [inline files/en-us/web/api/credentialscontainer/create/index.md:38]:
```suggestion
    Each of the following properties represents a _credential type_ being created. Only one of them can be specified:
```

**Review by wbamberg — APPROVED** [review]
👍 thank you @skyclouds2001 , this is much better.

**Pull request merged**